src/Tracking/TrackingGlue.cpp

```
@@ -47,23 +75,158 @@ TrackingGlue::OnTimer(const MoreData &basic, const DerivedInfo &calculated)
+        !IsInjectedSource(existing->source) &&
+        existing->valid)
+      continue;
+
+    FlarmTraffic built = online;
+    if (!SkyLinesTracking::FlarmTrafficBuilder::FillRelative(built, basic))
+      continue;
+
+    if (existing != nullptr) {
+      const FlarmId id = existing->id;
+      existing->Update(built);
+      existing->id = id;
```

Cppcheck (2.21.0)

[error] Rethrowing current exception with 'throw;', it seems there is no current exception to rethrow. If there is no current exception this calls std

(rethrowNoCurrentException)